src/metaschema/oscal_mapping-common_metaschema.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
NOTE: My review is based on the understanding that the relationships are applied solely to the information (set of requirements) provided by the `source` and `target` and NOT based on the options one would have to satisfy a requirement. In this case more restrictions means a larger set of requirements, and implicitly will trigger a larger set of actions to implement and more checks to verify.

CatalogA.control1 (Ac1) = “Requires TLS for transferring data, supporting only versions no less than v.1.2”

CatalogB.control2 (Bc2) = “ Requires TLS for transferring data”
CatalogB.Control3 (Bc3) = “ Requires TLS 1.2 or higher for PII transfer”
CatalogB.Control3 (Bc4) = “ Requires TLS 1.3 for payment transactions and related data ”

CatalogC.Control4 (Cc5) = “For sensitive information, data protection trough encryption is required. The cryptographic functions’ strength must be commensurable with the sensitivity of the information protected”

Here are the relations that can be expressed, based solely on the information (aka requirements) in the `source` vs `target` and NOT based on the set of options that would satisfy the set of requirements defined by the `source` relative to the set of requirements defined by the`target`.
1)	Ac1 _superset-of_ Bc2
2)	Ac1 _equivalent-to_ {Bc2, Bc3} 
3)	Ac1 _subset-of_ {Bc2, Bc4}
4)	Ac1 _xxxx_ Cc5. 
5)	Bc2 _yyyy_ Cc5
6)	{Bc2, Bc3, Bc4} _zzzz_ Cc5

Are all _xxxx_ , _yyyy_ and _zzzz_ identical with _superset-of_ because the sources have more restrictions? 
BUT _no-inference_ can be made like in the case 1) so without a different relation like _constrains_ with an antonym _relaxes_, I am not sure how else to make a distinction between 1) and 4), 5) 6)

The examples 4), 5) 6) are listed for the information purpose and are not meant to request support under this PR, but to set the foundation for future enhancements.

Reply:
Issue #1332 was created to address the cases where a control implements a more specific version of another control(s) requirements.